package.json

```diff
@@ -25,7 +25,12 @@
     "e2e-gen": "vitest src/__test__/e2e/general.test.ts",
     "e2e-kv": "vitest src/__test__/e2e/kv.test.ts",
     "e2e-ne": "vitest src/__test__/e2e/non-exportable.test.ts",
-    "e2e-sign": "vitest src/__test__/e2e/signing/signing.test.ts",
+    "e2e-sign": "npm run e2e-sign-determinism && npm run e2e-sign-evm-abi && npm run e2e-sign-evm-tx && npm run e2e-sign-solana && npm run e2e-sign-unformatted",
```

Comment:
`vitest src/__test__/e2e/signing` might accomplish the same result as calling all of the files individually (and it should be more future-proof)

Reply:
[074aaa8](https://github.com/GridPlus/gridplus-sdk/pull/467/commits/074aaa820cb7ae45a3de58c7cbafbbc6c1fb8287)